cmd/start.go

```diff
@@ -79,12 +81,18 @@ func startTimelock(cmd *cobra.Command) {
 		logs.Fatal().Msgf("value of poll-period not set: %s", err.Error())
 	}
 
-	tWorker, err := timelock.NewTimelockWorker(nodeURL, timelockAddress, callProxyAddress, privateKey, big.NewInt(fromBlock), pollPeriod, logs)
+	eventListenerPollPeriod, err := cmd.Flags().GetInt64("event-listener-poll-period")
+	if err != nil {
+		logs.Fatal().Msgf("value of poll-period not set: %s", err.Error())
```

Comment:
Agreed. I added "context" to the name of the new variable but left the existing one as is because it would break the "interface" and make the transition harder for current users.

We can discuss how to move forward in the next standup or tech discussion.